Tests/FluentAssertions.Specs/Types/TypeAssertionSpecs.cs

```diff
@@ -323,7 +259,7 @@ public void When_implemented_interface_type_it_succeeds()
         }
 
         [Fact]
-        public void When_an_unrelated_type_it_fails_with_a_useful_message()
+        public void When_an_unrelated_type_it_fails()
```

Comment:
🤔 This doesn't read very natural. Same for the other names.

Reply:
Although I agree, I only removed the `with_a_useful_message` part on existing tests.

I won't mind looking at better language in both test names and assertion messages of the existing 281 unit tests and 57 assertion methods, but I don't think we should add more this PR :)